ci/scripts/util_enable_core_dumps.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```

Comment:
How about removing this because this should be sourced?

```suggestion
```

Reply:
Good point!